### ISSUE TITLE
README Typo and misleading vars

### ISSUE THREAD

**Issue by mikeanth-dev** [ansible-collections/community.ciscosmb#50]
Problem #1:  There is a typo in the `README.md` file
-----------------------------------------------------------------------

Specifically, the typo is in the `ciscosmb_gather_facts.yml` playbook example, in the first task. The fully qualified module name specified is `comunTity.ciscosmb.facts` instead of `community.ciscosmb.facts`.
This made me think initially that there is something wrong with this module after copy-pasting the tasks to see if it works against my switches and being greeted by error messages.

Problem #2: The `gather_facts` variable specified in the inventory file is useless
---------------------------------------------------------------------------------------------------------

Furthermore, I believe that setting `gather_facts: no` in the `ciscosmb_inventory.yml` file is misleading as it does pretty much nothing, at least according to my testing. If I specify the variable in my inventory without also setting `gather_facts: false` in the playbook itself, Ansible still attempts to gather facts.

For the record, I am running ansible version 2.11.7
``` bash
mike@workstation:~$ ansible --version
ansible [core 2.11.7] 
    config file = /etc/ansible/ansible.cfg
    configured module search path = ['/home/mike/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
    ansible python module location = /usr/lib/python3/dist-packages/ansible
    ansible collection location = /home/mike/.ansible/collections:/usr/share/ansible/collections
    executable location = /usr/bin/ansible
    python version = 3.8.10 (default, Nov 26 2021, 20:14:08) [GCC 9.3.0]
    jinja version = 3.0.1
    libyaml = True
```

**Comment by mikeanth-dev** [comment]
@qaxi Opened a PR to address these issues.

**Comment by qaxi** [comment]
Solved by #51 
Thank you for PR.
